app/models/custom_button_set.rb

```diff
@@ -1,6 +1,11 @@
 class CustomButtonSet < ApplicationRecord
   acts_as_miq_set
 
+  before_save :update_children
+  def update_children
+    replace_children Rbac.filtered(CustomButton.find(set_data[:button_order]))
```

Comment:
@himdel what about `CustomButton.where(:id => [1,555572])` to don't throw exception when IDs don't exists ? I also prefer  `where`  because then  we can pass just query (`ActiveRecord::Relation`) to RBAC. (it is our long long term goal to pass only queries  to RBAC)

Reply:
Ah, makes sense, fixed :)